DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -706,6 +815,18 @@ void TopMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
       elePtEta_.at(iEle).denominator   -> Fill(electrons.at(iEle).pt(), electrons.at(iEle).eta() );
       eleEtaPhi_.at(iEle).denominator   -> Fill(electrons.at(iEle).eta(), electrons.at(iEle).phi() );
   }
+	//Menglei
+  if(enablePhotonPlot_){
+    for (unsigned int iPho=0; iPho<photons.size(); ++iPho){
+        if (iPho>=nphotons_) break;
+        phoPhi_.at(iPho).denominator  -> Fill(photons.at(iPho).phi());
```

Comment:
hi @mengleisun - this would need to be rebased to be merged - but otherwise some small comments

all the .at(iPho) here should be [iPho] - even better you can just do
for ( const auto & photon : photons) {...